eos-paygd/main.c

```diff
@@ -523,11 +527,12 @@ main (int   argc,
     {
       if (g_error_matches (error, EPG_SERVICE_ERROR, EPG_SERVICE_ERROR_NO_PROVIDER))
         {
-          /* This could mean the PAYG data has been erased; force a poweroff
-           * after 10 minutes. See https://phabricator.endlessm.com/T27581
+          /* This could mean the PAYG data has been erased; force a poweroff.
+           * See https://phabricator.endlessm.com/T27581
            */
-          g_warning ("Initiating 10 minute shutdown timer due to error: %s", error->message);
-          timeout_id = g_timeout_add_seconds (10 * 60, payg_sync_and_poweroff, NULL);
+          g_warning ("Provider failure, shutting down in %d minutes: %s",
+                     POWEROFF_TIMEOUT_ERROR, error->message);
+          timeout_id = g_timeout_add_seconds (POWEROFF_TIMEOUT_ERROR * 60, payg_sync_and_poweroff, NULL);
```

Comment:
What is the rationale for increasing this from 10 to 20? (I think it's probably OK.)

Reply:
Just making all error paths have the same timeout length. Maybe this was intentionally shorter before because it was easier to damage the provider data storage? Now the storage is protected by the LSM (even when using the eMMC backend, to some extent), so I think it is fine to have the same error timeout.